targets/targets.json

```diff
@@ -2306,42 +2306,87 @@
     },
     "SDP_K1": {
         "inherits": [
-            "MCU_STM32"
+            "Target"
```

Comment:
What's the reason for not inherit but duplicate (at least that is the question) ?

Reply:
Changing the board inheritance from STM32 family to Target to avoid dependence on MCU_STM32 definition. If there are changes to MCU_STM32 definition in the future, this ensures SDP_K1 attributes remain unchanged.